cmds/core/cp/cp_test.go

```diff
@@ -255,10 +326,11 @@ func TestCpSymlink(t *testing.T) {
 
 	t.Run("follow-symlink", func(t *testing.T) {
 		defer resetFlags()
+		var out bytes.Buffer
 		flags.noFollowSymlinks = false
 
 		dst := filepath.Join(tempDir, "dst-follow")
-		if err := cpArgs([]string{newName, dst}); err != nil {
+		if err := run([]string{newName, dst}, false, false, false, false, false, &out); err != nil {
```

Comment:
This right here outlines one of the biggest gripes I have with Go: lack of type enumerations, see what I mean: 
https://go.dev/play/p/iK4-s1INt7I

but it would sure be nice if we had a call that did not have 5 boolean opts in a row. It's hard to figure out what this could be.

This leads me to wonder about something like this:
https://go.dev/play/p/ikpClk3IWmg
Which is a bit more descriptive, albeit wordier?